WooCommerce/WooCommerceTests/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUpViewModelTests.swift

```diff
@@ -100,4 +117,102 @@ final class InPersonPaymentsCashOnDeliveryPaymentGatewayNotSetUpViewModelTests:
         let expectedTitle = "Failed to enable Pay in Person. Please try again later."
         assertEqual(expectedTitle, notice.title)
     }
+
+    // MARK: - Analytics tests
+    func test_skip_tapped_logs_onboarding_step_skipped_event() throws {
```

Comment:
Should we rename these tests to something more descriptive? As there's multiple asserts within each that represents different parameters of the expected event, maybe something like `test_received_event_when_skipTapped_then_logs_onboarding_step_skipped` could be easier to understand at first glance.

Reply:
We could... but I don't think that's particularly easier to understand. I'd actually suggest making it shorter to make it more readable, e.g. `test_skipTapped_tracks_correct_analytics_event`, but that is less specific.